src/converter.js

```diff
@@ -375,6 +375,34 @@ var resolveRef = function(oai, ref) {
   return recursive(oai, 1)
 }
 
+/**
+ * Returns mock data based on the passed data type
+ *
+ * @param  {string} type the data type we to mock
+ * @param  {string} key associated with value to make more customized returns
+ * @return {any}
+ */
+var getMockData = function(type) {
+  switch (type) {
+    case 'STRING':
+      return 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+    case 'INTEGER':
+      return Math.floor(Math.random()*100);
+    case 'NUMBER':
+      return Math.random().toFixed(5)*100;
+    case 'BOOLEAN':
+      return true;
+    case 'ARRAY':
+      return ['lorem', 2, true, {key: 'value'}];
+    case 'OBJECT':
+      return ({
+        key: 'value'
+      });
+  }
+  return '';
+}
```

Comment:
You doesn't cover following types:
* `integer`
* `null`

I couldn't ignore type restrictions:
* **string**: `pattern`, `format`, `enum`, `minLength` and `maxLength`
* **number** and **integer**: `minimum`, `maximum`, `exclusiveMaximum`, `exclusiveMinimum`, `multipleOf`, `enum`
* **array**: `items`, `minItems`, `maxItems`, `uniqueItems`, `additionalItems` (`items` types should be handled recursively)
* **object**: `required`, `properties`, `propertyNames`, `minProperties`, `maxProperties`, `patternProperties`

Reply:
Use `example`, if it's defined